sharding-distribution/sharding-scaling-distribution/src/main/resources/conf/log4j.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+log4j.rootLogger=DEBUG, root
+log4j.appender.root=org.apache.log4j.ConsoleAppender
+log4j.appender.root.layout=org.apache.log4j.PatternLayout
+log4j.appender.root.layout.ConversionPattern=%d [%t] %p %l - %m%n
```

Comment:
I understand ShardingSphere use logback as log component, do we need log4j?

Reply:
Yes, we should transform the log4j to logback, a new issue may be submitted.

Reply:
I think we need to use logback replace log4j, we can change in other PR.
This Pr for adding sharding-scaling-distribution module only, What do you think about?

Reply:
OK, I understand.

Reply:
Yes, I agree with you @KomachiSion .

Reply:
@dongzl , If you think the PR can be merged, can you approve it?